account_bank_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0
+
+from odoo.tests import common
+from odoo import fields
+
+
+class TestAccountBankStatementImportMoveLine(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountBankStatementImportMoveLine, cls).setUpClass()
+        cls.account_type = cls.env['account.account.type'].create({
+            'name': 'Test Account Type'})
+        cls.a_receivable = cls.env['account.account'].create({
+            'code': 'TAA',
+            'name': 'Test Receivable Account',
+            'internal_type': 'receivable',
+            'user_type_id': cls.account_type.id,
+        })
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'Test Partner 2',
+            'parent_id': False,
+            'account_id': cls.a_receivable.id,
```

Comment:
There is not exist a field "account_id" in res_partner model. Is It necessary that field?